templates/stub.aws.example.yml

```diff
@@ -47,7 +51,7 @@ jobs:
   - (( merge ))
   - name: elastic
     static_ips:
-    - VAR_ELK_IP  # Specify an elastic IP for the router. Resources, proxied by the router, will be available through this IP. They include: kibana, elasticsearch, cluster_monitor, ingestor and syslog_server.
+    - (( grab meta.aws.eip )) 
```

Comment:
It seems that this EIP, which attached to `ls-router` job, won't work at all. We usually deploy Logsearch to private AWS subnet, that mean that instances in this subnet can access Internet by NAT and you cannot associate EIP with instances in NAT subnet, EIP can be associated only with instances attached to Internet gateway.